README.md

```diff
@@ -49,7 +49,7 @@ ACCESS_KEY="<base-64 Access Key string created from step 2>"
 
 REPOSITORY_ID="<Repository ID from the 'Repository Administration' page>"
 
-AUTHORIZATION_TYPE="AccessKey"
+API_ENVIRONMENT_UNDER_TEST="CloudClientCredentials"
```

Comment:
Better to leave the name as `AUTHORIZATION_TYPE` since the sample project isn't meant to be a "test", but a sample of how the client lib can be used in a real application